contracts/OpenSTValue.sol

```diff
@@ -149,13 +153,18 @@ contract OpenSTValue is OpsManaged, Hasher {
     constructor(
         uint256 _chainIdValue,
         EIP20Interface _eip20token,
-        address _registrar)
+        address _registrar,
+        uint256 _blockTime)
```

Comment:
Block time of what?

Reply:
changed to `_valueChainBlockGenerationTime `